src/NuGet.Core/NuGet.Protocol/Model/IPackageSearchMetadata.cs

```diff
@@ -37,6 +37,8 @@ public interface IPackageSearchMetadata
         bool IsListed { get; }
         bool PrefixReserved { get; }
 
+        bool? HasReadme { get; }
```

Comment:
What's the benefit of nullable here?

Reply:
Happy to hear more thoughts on this, but my thought was that I could use the null value to indicate that we are unable to determine if a readme is available for the package. 

I'm not super happy with this solution, I could change it to an ENUM which makes it clear what the values are but even then, I'm not sure that it makes sense to have that be part of the model.